### ISSUE TITLE
Invalid URL 'None': No schema supplied.

### ISSUE THREAD

**Issue by bollerdominik** [BreadedChickenator/Cross-Seed-AutoDL#7]
Hi
When running this it starts downloading a few torrents successfully but then always fails at the same place:

```
Traceback (most recent call last):
  File "CrossSeedAutoDL.py", line 232, in <module>
    main()
  File "CrossSeedAutoDL.py", line 105, in main
    found, announceDownloadings = findMatchingTorrent(returnedJSON, pathSize, path)
  File "CrossSeedAutoDL.py", line 201, in findMatchingTorrent
    downloadTorrent(downloadURL, listingTitle)
  File "CrossSeedAutoDL.py", line 214, in downloadTorrent
    response = requests.get(downloadURL, stream=True)
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\sessions.py", line 516, in request
    prep = self.prepare_request(req)
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\sessions.py", line 449, in prepare_request
    p.prepare(
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\models.py", line 314, in prepare
    self.prepare_url(url, params)
  File "C:\Users\testuser\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\models.py", line 388, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL 'None': No schema supplied. Perhaps you meant http://None?
```

**Comment by bollerdominik** [comment]
Just saw a related PR #6 .

**Comment by BC44** [comment]
Fixed in 48451034de97fc310478b7c21c1ef590e0372037
